rotkehlchen/api/rest.py

```diff
@@ -2224,6 +2224,7 @@ def get_ethereum_airdrops(self) -> dict[str, Any]:
             data = check_airdrops(
                 addresses=self.rotkehlchen.chains_aggregator.accounts.eth,
                 data_dir=self.rotkehlchen.data_dir,
+                database=self.rotkehlchen.data.db,
```

Comment:
Can't check as still mobile, but doesn't the database also contain the data directory or not? If it does perhaps you can remove it from above as an argument.